src/federationparams.cpp

```diff
@@ -102,7 +102,11 @@ std::unique_ptr<CFederationParams> CreateFederationParams(const TAPYRUS_OP_MODE
 {
     gArgs.SelectConfigNetwork(TAPYRUS_MODES::GetChainName(mode));
 
-    const int networkId = gArgs.GetArg("-networkid", TAPYRUS_MODES::GetDefaultNetworkId(mode));
+    int64_t nid;
+    bool inRange = gArgs.IsGetArgInRange("-networkid", 1, 4294967295,  TAPYRUS_MODES::GetDefaultNetworkId(mode), nid);
```

Comment:
`UINT_MAX` can be used instead of `4294967295`?

Reply:
oh yes!